playwright/_impl/_api_structures.py

```diff
@@ -195,3 +196,150 @@ class FrameExpectResult(TypedDict):
     matches: bool
     received: Any
     log: List[str]
+
+
+# see http://www.softwareishard.com/blog/har-12-spec/
```

Comment:
Something is not quite right as I'm getting when running  `./scripts/update_api.sh` (yet I don't see how this is different from our use of `ResourceTiming` typed dict:

```
(env) rwoll:playwright-python rosswollman$ ./scripts/update_api.sh
Generating playwright/sync_api/_generated.py
Parameter type mismatch in Route.fulfill(response=): documented as Union[APIResponse, HARResponse, NoneType], code has Union[APIResponse, {status: float, statusText: str, httpVersion: str, cookies: List[{name: str, value: str, path: Union[str, NoneType], domain: Union[str, NoneType], expires: Union[str, NoneType], httpOnly: Union[bool, NoneType], secure: Union[bool, NoneType], sameSite: Union[str, NoneType], comment: Union[str, NoneType]}], headers: List[{name: str, value: str, comment: Union[str, NoneType]}], content: {size: float, compression: Union[float, NoneType], mimeType: str, text: Union[str, NoneType], encoding: Union[str, NoneType], comment: Union[str, NoneType]}, redirectURL: str, headersSize: float, bodySize: float, comment: Union[str, NoneType]}, NoneType]
```

Reply:
struct ported from: https://github.com/microsoft/playwright/commit/e00a26a11de55446fae5edc806a7236af725c3f8?diff=unified#diff-3d5c3176db8e25c20b575634893a81a93ca1781de32879502bafc68b462be7afR1